File: boot/acp-spring-boot-starter-ftp/src/main/java/pers/acp/spring/boot/ftp/conf/FtpServerConfiguration.java
package pers.acp.spring.boot.ftp.conf;

import pers.acp.ftp.conf.FtpListener;

import java.util.ArrayList;
import java.util.List;

/**
 * Ftp 服务端配置
 *
 * @author zhang by 21/06/2019
 * @since JDK 11
 */
public class FtpServerConfiguration {

    public List<FtpListener> getListeners() {
        return listeners;
    }

    public void setListeners(List<FtpListener> listeners) {
        this.listeners = listeners;
    }

    /**
     * Ftp服务监听配置列表
     */
    private List<FtpListener> listeners = new ArrayList<>();

}

File: boot/acp-spring-boot-starter-ftp/src/main/kotlin/pers/acp/spring/boot/ftp/conf/FtpServerConfiguration.kt
package pers.acp.spring.boot.ftp.conf

import pers.acp.ftp.conf.FtpListener

/**
 * Ftp 服务端配置
 *
 * @author zhang by 21/06/2019
 * @since JDK 11
 */
class FtpServerConfiguration {

    /**
     * Ftp服务监听配置列表
     */
    var listeners: MutableList<FtpListener> = mutableListOf()

}
